### ISSUE TITLE
Module commerce: impossible to order with layout 9 and connect layout

### ISSUE THREAD

**Issue by lsmod** [Xtraball/SiberianCMS#421]
module commerce:
(siberian 4.12.9)

impossible to order with layout 9 and connect layout and module commerce

When clicking on next in "My information page" (after added product to cart then click proceed), module open a blank page then return app first page (without saving the order)

On the blank page this error get logged :
```
Error: [ngModel:datefmt] app.bundle-min.js?version=4.12.9.1504616546:8 http://errors.angularjs.org/1.5.3/ngModel/datefmt?p0=1990-04-12T22%3A00%3A00.000Z
    at app.bundle-min.js?version=4.12.9.1504616546:7
    at Array.<anonymous> (app.bundle-min.js?version=4.12.9.1504616546:9)
    at app.bundle-min.js?version=4.12.9.1504616546:11
    at m.$digest (app.bundle-min.js?version=4.12.9.1504616546:9)
    at m.$apply (app.bundle-min.js?version=4.12.9.1504616546:9)
    at g (app.bundle-min.js?version=4.12.9.1504616546:8)
    at x (app.bundle-min.js?version=4.12.9.1504616546:8)
    at XMLHttpRequest.v.onload (app.bundle-min.js?version=4.12.9.1504616546:8)
```

**Comment by bluesnap-rock** [comment]
Hi Guys, how long before there is a fix for the mcommerce its not working on some layouts? thanks guys.

**Comment by Slicit** [comment]
The issue is fixed in 4.12.10.

**Comment by lsmod** [comment]
Nice justed tested with layout9 and homepage connect it's working fine!
